README.md

```diff
@@ -120,9 +120,13 @@ These options are documented below:
 | `--read-only` | Enable or disable read only mode | `bool` | `false` |
 | `--web-dir` | Dashboard web resources directory | `string` | `""` |
 | `--logout-url` | If set, enables logout on the frontend and binds the logout button to this url | `string` | `""` |
+| `--namespace` | If set, limits the scope of resources watched to this namespace only | `string` | `""` |
```

Comment:
Let's hold off on adding the readme update until the frontend piece is in place. We know some of the experience will be broken util that's complete.

Reply:
References to `--namespace` have been removed.